_data/swanlake-latest/metadata.json

```diff
@@ -1,21 +1,20 @@
 {
-	"version":"swan-lake-alpha5",
-	"short-version":"slalpha5",
-	"display-version":"Swan Lake Alpha5",
-	"release-date":"2021-04-24",
-	"windows-installer":"ballerina-windows-installer-x64-swan-lake-alpha5.msi",
-	"windows-installer-size":"142mb",
-	"linux-installer":"ballerina-linux-installer-x64-swan-lake-alpha5.deb",
-	"linux-installer-size":"206mb",
-	"macos-installer":"ballerina-macos-installer-x64-swan-lake-alpha5.pkg",
-	"macos-installer-size":"237mb",
-	"rpm-installer":"ballerina-linux-installer-x64-swan-lake-alpha5.rpm",
-	"rpm-installer-size":"217mb",
+	"version":"swan-lake-beta1",
+	"short-version":"slbeta1",
+	"display-version":"Swan",
```

Comment:
Are we planning to keep the `display-version` as` Swan`? Shouldn't this be `Swan Lake Beta1`?

Reply:
Seems a bug in installer generation. Fixed it here for now.